lib/components/tabs/tabview.coffee

```diff
@@ -110,7 +110,10 @@ module.exports = class KDTabView extends KDScrollView
       minWidth : minHandleWidth
 
     newTabHandle.on 'HandleIndexHasChanged',  @bound 'resortTabHandles'
-    newTabHandle.closeHandler?.on 'click', => @handleCloseAction paneInstance
+
+    unless paneInstance.isDetached
```

Comment:
I think not. Because `paneInstance` has `parent` every time when `addPane` is called.
So `newTabHandle` will bind `click` event more than one.

Can we make a session about this on slack or hangouts when you're available?